containerfixture/containerfixture.go

```diff
@@ -0,0 +1,118 @@
+package containerfixture
+
+import (
+	_ "embed"
+	"fmt"
+	"log"
+	"math/rand"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+//go:embed containerfixture.yaml
+var manifest string
+
```

Comment:
Do these consts need to be exported? If not - start with lowercase letters.

Reply:
Currently not, but a Splunk test will need the Username and Password detaills, unless those are sent to the function.
I'd say the other way around is more suitable (them being imported FROM the Splunk package), but that'll create circular dependencies.

Reply:
pass them as parameters, or otherwise us a netrc  file as suggested below.

In general if we can use files or env vars to avoid having the code deal with credentials its almost always better.

Reply:
How is a generic fixture package depending on a Specific "Splunk" package that is using it "more suitable"?

Reply:
It does not depend on a Splunk package, but if the function will get generlized, and will get used on the Splunk test, it might just get the`username` and `password` as parameters, without importing them.   
anyway, I already said below I'll check the netrc/parameters approach

Reply:
using a .netrc file